integrations/java-iceberg-1.5/openhouse-java-runtime/build.gradle

```diff
@@ -0,0 +1,83 @@
+plugins {
+  id 'openhouse.java-minimal-conventions'
+  id 'openhouse.maven-publish'
+  id 'com.github.johnrengelman.shadow' version '7.1.2'
+}
+
+configurations {
+  fatJarPackagedDependencies {
+    // Following exclusions are not needed during runtime
+    // and often cause conflict with existing classpath.
+    exclude(group: 'org.slf4j') // logging libraries
+    exclude(group: 'org.apache.log4j')
+    exclude(group: 'org.apache.logging.log4j')
+    exclude(group: 'org.mapstruct')
+    exclude(group: 'io.micrometer') // not used in client
+    exclude(group: 'ch.qos.logback')
+  }
+  shadow.extendsFrom implementation
+}
+
+ext {
+  icebergVersion = '1.5.2'
+  sparkVersion = '3.5.2'
+  springVersion = '2.7.8'
+  hadoopVersion = '2.10.0'
+}
+
+dependencies {
+  compileOnly project(':client:secureclient')
```

Comment:
you can declare dependency on java moduel for 1.2 if the code is not changing. basically:

```
dependencies {
         api (java-rntime_iceberg_1.2) {
                  transitive=false
        }
}
```